src/frontend/src/pages/AdminToolsPage/AdminToolsPage.tsx

```diff
@@ -12,18 +12,63 @@ import AdminToolsFinanceConfig from './AdminToolsFinanceConfig';
 import TeamsTools from './TeamsTools';
 import AdminToolsBOMConfig from './AdminToolsBOMConfig';
 import AdminToolsProjectsConfig from './AdminToolsProjectsConfig';
+import { useState } from 'react';
+import NERTabs from '../../components/Tabs';
+import { routes } from '../../utils/routes';
 
 const AdminToolsPage: React.FC = () => {
   const currentUser = useCurrentUser();
+  const [tabIndex, setTabIndex] = useState<number>(0);
+
+  const isUserHead = isHead(currentUser.role);
+  const isUserAdmin = isAdmin(currentUser.role);
+  const isUserFinanceLead = currentUser.isAtLeastFinanceLead;
+
+  const defaultTab = isUserAdmin || isUserHead ? 'user-management' : 'finance-configuration';
+
+  const tabs = [];
+
+  if (isUserHead || isUserAdmin) {
+    tabs.push({ tabUrlValue: 'user-management', tabName: 'User Management' });
+    tabs.push({ tabUrlValue: 'project-configuration', tabName: 'Project Configuration' });
+  }
+  if (isUserAdmin || isUserFinanceLead) {
+    tabs.push({ tabUrlValue: 'finance-configuration', tabName: 'Finance Configuration' });
+  }
+  if (isUserAdmin) {
+    tabs.push({ tabUrlValue: 'miscellaneous', tabName: 'Miscellaneous' });
+  }
+
+  const showUserManagement = () => {
```

Comment:
you can call it UserManagementTab, then call it like a component: `<UserManagementTab/>` below